lib/agent/providers/network/windows.js

```diff
@@ -141,33 +141,35 @@ exports.parse_access_points_list_netsh = function(out) {
 
   var get_values = function(str) {
     var res = {};
+    var idx = 0;
     str.split('\n').forEach(function(line) {
       if (line.toString().trim() == '')
         return;
 
       var split = line.split(': '),
-          key   = split[0].trim(),
+          key   = idx.toString(),
           val   = split[1] ? split[1].trim() : null;
 
       if (key) res[key] = val; 
+      idx++;
     })
     return res;
   }
 
   var build_ap = function(base, router) {
     var obj = {
-      ssid        : base['SSID'].toString().trim() == '' ? '(Unknown)' : base['SSID'],
-      security    : base['Authentication'] != 'Open' ? base['Authentication'] : null,
-      mac_address : router['BSSID']
+      ssid        : base['0'].toString().trim() == '' ? '(Unknown)' : base['0'],
```

Comment:
Could you provide enumerations for the parts indexes?

For instance:
```
SSID = '0'
ssid : base[SSID].toString().trim() == '' ? '' ...
```

as opposed to

```
ssid : base['0'].toString().trim() == '' ? '' ...
```